backend/dataall/base/utils/api_handler_utils.py

```diff
@@ -26,6 +26,14 @@
 ALLOWED_ORIGINS = os.getenv('ALLOWED_ORIGINS', '*')
 
 
+def redact_creds(event):
+    if 'headers' in event and 'Authorization' in event['headers']:
```

Comment:
nit: probably something like `if event.get('headers', {}).get('Authorization') ...` is cleaner